### PR TITLE
[v0.15.x] Backport pending PRs from develop

### PR DESCRIPTION
**Pull request by codablock**
Backport PRs with label [backport-candidate-15.x](https://github.com/dashpay/dash/pulls?q=is%3Apr+label%3Abackport-candidate-15.x+is%3Aclosed)

Also updates release-notes.md

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -223,6 +223,15 @@ modules were reorganized in separate folders to make navigation through code a b
...
 
+- [`1d9adbe639`](https://github.com/dashpay/dash/commit/1d9adbe639) Replace generic CScopedDBTransaction with specialized CEvoDBScopedCommitter (#3292)
+- [`8fd486c6bc`](https://github.com/dashpay/dash/commit/8fd486c6bc) Translations 2020-01 (#3192)
+- [`3c54f65278`](https://github.com/dashpay/dash/commit/3c54f65278) Bump copyright year to 2020 (#3290)
+- [`35d28c7485`](https://github.com/dashpay/dash/commit/35d28c7485) Update man pages (#3291)
+- [`203cc90778`](https://github.com/dashpay/dash/commit/203cc90778)  trivial: adding SVG and high contrast icons  (#3209)
+- [`e875d4925a`](https://github.com/dashpay/dash/commit/e875d4925a) Define defaultTheme and darkThemePrefix as constants and use them instead of plain strings (#3288)
+- [`1d203b422c`](https://github.com/dashpay/dash/commit/1d203b422c) Bump PROTOCOL_VERSION to 70216 (#3287)
+- [`b84482ac57`](https://github.com/dashpay/dash/commit/b84482ac57) Let regtest have its own qt settings (#3286)
+- [`1c885bbedf`](https://github.com/dashpay/dash/commit/1c885bbedf) Only load valid themes, fallback to "Light" theme otherwise (#3285)
```
Comment (UdjinM6) [inline doc/release-notes.md]: nit: not sure why, but my short commit hashes produced via `git log --oneline --no-merges master..HEAD` were slightly shorter...

Comment (UdjinM6) [reply doc/release-notes.md]: 4ff5cdebb

Comment (codablock) [reply doc/release-notes.md]: Yeah noticed this as well, my assumption is that I have a local conflict when shorter hashes are used

Comment (codablock) [reply doc/release-notes.md]: @UdjinM6 your commit only touched the part in [] but not the link itself. Is this what you wanted?

Comment (UdjinM6) [reply doc/release-notes.md]: ooops 🙈 187250863

Comment (UdjinM6) [reply doc/release-notes.md]: would probably make sense to squash all fixes into one `Update release-notes.md` commit

Comment (codablock) [reply doc/release-notes.md]: done

**Comment by UdjinM6** [comment]
FYI: dropped `backport-candidate-15.x` labels from corresponding PRs

**Pull request merged**